wagtailvideos/jinja2tags.py

```diff
@@ -1,22 +1,14 @@
 from __future__ import absolute_import, unicode_literals
 
-import mimetypes
-
 from jinja2.ext import Extension
 
-from django import template
-from django.forms.widgets import flatatt
-from django.utils.text import mark_safe
-
+from .models import Video
 
-def video(video, attrs):
-    if not video:
-        return ''
 
-    if attrs:
-        return video.video_tag(attrs)
-    else:
-        return video
+def video(video, attrs='preload controls'):
+    if type(video) != Video:
```

Comment:
The pythonic way of doing this is `if isinstance(video, Video):`, which also handles `video` being a subclass of `Video`.